models/offer.go

```diff
@@ -198,9 +205,17 @@ func GetAvailableOffers(db runner.Connection, playerID, gameID string, t time.Ti
 		return map[string][]*OfferToReturn{}, nil
 	}
 
-	playerOffersByOfferTemplateID := map[string]*Offer{}
+	playerOffersByOfferTemplateKey := map[string]*Offer{}
 	for _, o := range playerOffers {
-		playerOffersByOfferTemplateID[o.OfferTemplateID] = o
+		playerOffersByOfferTemplateKey[o.OfferTemplateKey] = o
```

Comment:
A player could have several offers with se same OfferTemplateKey if the template was updated several times, for example.
```
playerOffersByOfferTemplateKey := map[string][]*Offer{}
```